python/labbox_ephys/import_to_workspace.py

```diff
@@ -36,6 +36,13 @@ def _get_sortings_from_subfeed(sortings_subfeed: kp.Subfeed):
                 for sid in a.get('sortingIds', []):
                     if sid in le_sortings:
                         del le_sortings[sid]
+            elif a.get('type', '') == 'DELETE_SORTINGS_FOR_RECORDINGS':
```

Comment:
When a recording(s) is deleted in the GUI, we also dispatch a DELETE_SORTINGS_FOR_RECORDINGS which also deletes the sortings that are associated with the deleted recording(s).